src/ui/state/computed/glyph-computed.js

```diff
@@ -2,69 +2,99 @@ import { getAllGlyphs } from '../../../export/font-loader.js';
+  (unicode >= 0x41 && unicode <= 0x5a) || (unicode >= 0x61 && unicode <= 0x7a);
 
 const GLYPH_CATEGORIES = [
-  { name: 'Special', filter: (g) => SPECIAL_UNICODES.includes(g.unicode) },
-  { name: 'Basic Glyphs', filter: (g) => isBasicLetter(g.unicode) },
-  { name: 'Numerals', filter: (g) => isNumeral(g.unicode) },
+  { name: 'Special', filter: (glyph) => SPECIAL_UNICODES.includes(glyph.unicode) },
+  { name: 'Basic Glyphs', filter: (glyph) => isBasicLetter(glyph.unicode) },
+  { name: 'Numerals', filter: (glyph) => isNumeral(glyph.unicode) },
   {
     name: 'Punctuation & Others',
-    filter: (g) => {
-      const u = g.unicode;
-      if (u === 0x20 || u === 0x0d) return false;
-      if (SPECIAL_UNICODES.includes(u)) return false;
-      if (isBasicLetter(u)) return false;
-      if (isNumeral(u)) return false;
-      if (isAccentedLetter(u)) return false;
-      if (isSymbol(u)) return false;
-      return u < 0xe000;
+    filter: (glyph) => {
+      const unicode = glyph.unicode;
+      if (unicode === 0x20 || unicode === 0x0d) {
+        return false;
+      }
+      if (SPECIAL_UNICODES.includes(unicode)) {
+        return false;
+      }
+      if (isBasicLetter(unicode)) {
+        return false;
+      }
+      if (isNumeral(unicode)) {
+        return false;
+      }
+      if (isAccentedLetter(unicode)) {
+        return false;
+      }
+      if (isSymbol(unicode)) {
+        return false;
+      }
+      return unicode < 0xe000;
     },
   },
   { name: 'Accents', filter: (g) => isAccentedLetter(g.unicode) },
   { name: 'Symbols', filter: (g) => isSymbol(g.unicode) },
 ];
 
 function getNumeralSortOrder(unicode) {
-  if (unicode >= 0x30 && unicode <= 0x39) return unicode - 0x30;
-  if (unicode === 0x2070) return 10;
-  if (unicode === 0xb9) return 11;
-  if (unicode === 0xb2) return 12;
-  if (unicode === 0xb3) return 13;
-  if (unicode >= 0x2074 && unicode <= 0x2079) return 14 + (unicode - 0x2074);
-  if (unicode >= 0x2080 && unicode <= 0x2089) return 20 + (unicode - 0x2080);
-  if (unicode === 0xbc) return 30;
-  if (unicode === 0xbd) return 31;
-  if (unicode === 0xbe) return 32;
-  if (unicode >= 0x2150 && unicode <= 0x218f) return 33 + (unicode - 0x2150);
-  return 100 + unicode;
+  switch (true) {
+    case unicode >= 0x30 && unicode <= 0x39:
+      return unicode - 0x30;
+    case unicode === 0x2070:
+      return 10;
+    case unicode === 0xb9:
+      return 11;
+    case unicode === 0xb2:
+      return 12;
+    case unicode === 0xb3:
+      return 13;
+    case unicode >= 0x2074 && unicode <= 0x2079:
+      return 14 + (unicode - 0x2074);
+    case unicode >= 0x2080 && unicode <= 0x2089:
+      return 20 + (unicode - 0x2080);
+    case unicode === 0xbc:
+      return 30;
+    case unicode === 0xbd:
+      return 31;
+    case unicode === 0xbe:
+      return 32;
+    case unicode >= 0x2150 && unicode <= 0x218f:
+      return 33 + (unicode - 0x2150);
+    default:
+      return 100 + unicode;
+  }
```

Comment:
Any reason for the `switch(true)` pattern here? The if-chain felt a bit more natural for this (and we don't need 3x the lines) ... I think switch usually shines when matching one value against many constants.